src/tests.rs

```diff
@@ -235,10 +237,42 @@ d
         assert_eq!(
             hotkeys,
             vec!(
-                Hotkey::new(evdev::Key::KEY_D, vec![], String::from("d")),
                 Hotkey::new(evdev::Key::KEY_C, vec![], String::from("c")),
                 Hotkey::new(evdev::Key::KEY_A, vec![], String::from("a")),
                 Hotkey::new(evdev::Key::KEY_B, vec![], String::from("b")),
+                Hotkey::new(evdev::Key::KEY_D, vec![], String::from("d")),
+            )
+        );
+        Ok(())
+    }
+    #[test]
+    fn test_include_and_unbind() -> std::io::Result<()> {
+        let setup = TestPath::new("/tmp/swhkd-test-file8");
```

Comment:
why do we create temporary files on disk every time? why not use a memory backed file that lives in ram which is automatically `free`'d when tests are over?

This can be done with fcntl, memfd_create and other libc syscalls.